noir-projects/noir-contracts/contracts/avm_test_contract/src/main.nr

```diff
@@ -140,6 +141,19 @@ contract AvmTest {
     //     context.contract_call_depth()
     // }
 
+    #[aztec(public-vm)]
+    fn emit_unencrypted_log() {
+        context.emit_unencrypted_log(/*event_selector=*/ 5, /*message=*/ [10, 20, 30]);
+        context.emit_unencrypted_log(/*event_selector=*/ 8, /*message=*/ "Hello, world!");
+        // FIXME: Try this once Brillig codegen produces uniform bit sizes for LT
+        // FIXME: TagCheckError: Tag mismatch at offset 22, got UINT64, expected UINT32
+        // let s: CompressedString<1,13> = CompressedString::from_string("Hello, world!");
+        // context.emit_unencrypted_log(/*event_selector=*/ 10, /*message=*/ s);
+    }
+
+    #[aztec(public-vm)]
+    fn emit_unencrypted_log_compressed_string() {}
```

Comment:
can you throw a comment on the stub, else we can remove

Reply:
Actually I'm now doing this above so no need. I'll remove.